src/core/StaticIcon/StaticIcon.tsx

```diff
@@ -49,18 +57,18 @@ export class StaticIcon extends Component<StaticIconProps> {
       highlightColor,
       className,
       ...passProps
-    } = withSuomifiDefaultProps(this.props);
+    } = this.props;
     const { ariaLabel } = this.props;
 
     if (icon !== undefined) {
       return (
         <StyledSuomifiStaticIcon
           {...passProps}
-          className={classnames(baseClassName, className)}
           {...(!!highlightColor
             ? { highlightColor }
             : { highlightColor: tokens.colors.accentBase })}
           icon={icon}
+          className={getClassNames(this.props)}
```

Comment:
Was there a certain benefit for having the classnames done in separate function?

Reply:
Removed and logic now in JSX.